build.cmd

```diff
@@ -8,22 +8,22 @@ IF ERRORLEVEL 1 CALL:FAILED_CASE
 IF ERRORLEVEL 1 CALL :DEFAULT_CASE
 
 :Prepare_CASE
-    dotnet build Xamarin.Android.sln -t:Prepare -nr:false
+    dotnet msbuild Xamarin.Android.sln -t:Prepare -nr:false
+    dotnet msbuild external\Java.Interop\Java.Interop.sln -t:Prepare -nr:false
     GOTO END_CASE
 :PrepareExternal_CASE
     dotnet build Xamarin.Android.sln -t:PrepareExternal -nr:false
     GOTO END_CASE
 :Build_CASE
     dotnet-local.cmd build Xamarin.Android.sln  -nr:false
-    dotnet-local.cmd build tools/xabuild/xabuild.csproj -nr:false
     GOTO END_CASE
 :Pack_CASE
     dotnet-local.cmd build  Xamarin.Android.sln -t:PackDotNet -nr:false
     GOTO END_CASE
 :DEFAULT_CASE
-    dotnet build Xamarin.Android.sln -t:Prepare -nr:false
+    dotnet msbuild Xamarin.Android.sln -t:Prepare -nr:false
+    dotnet msbuild external\Java.Interop\Java.Interop.sln -t:Prepare -nr:false
```

Comment:
How does this relate to:

https://github.com/xamarin/xamarin-android/blob/main/build-tools/scripts/PrepareWindows.targets

Should we put the step in this `.targets` file or in the `.cmd` file?

Reply:
There is also already a [`PrepareJavaInterop` target](https://github.com/xamarin/xamarin-android/blob/042035117e4550bae313336d876bae9a78e1bff0/build-tools/scripts/DotNet.targets#L18) that is used by the default Windows CI build case of:

 * dotnet build -t:Prepare Xamarin.Android.sln
 * dotnet build -t:BuildDotNet,PackDotNet Xamarin.Android.sln

Maybe we could instead override the default Build target for Xamarin.Android.sln on Windows to run `BuildDotNet` (which depends on `PrepareJavaInterop`) ?